src/cmake/config.cmake.in

```diff
@@ -29,7 +29,9 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 include_directories(@PACKAGE_INCLUDE_INSTALL_DIR@)
 
-if (@QTAWSIOT_STATIC_LIB@)
-    message(STATUS "Using statically linked QtAwsIoT library")
-    add_definitions(-DQTAWSIOT_STATIC_LIB)
+if (@BUILD_SHARED_LIBS@)
+    message(STATUS "Using dynamic linked QtAwsIoT library")
```

Comment:
I tried to avoid the exact names of the library in the config.cmake.in file so that it is easily reusable. Could you use @PROJECT_NAME@ (IIRC) instead of QtAwsIoT?